sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastHelper.scala

```diff
@@ -77,5 +79,18 @@ object GpuBroadcastHelper {
         throw new IllegalStateException(s"Invalid broadcast batch received $t")
     }
   }
+
+  /** Given a broadcast relation return an RDD of that relation. */
+  def asRDD(sc: SparkContext, broadcast: Broadcast[Any]): RDD[ColumnarBatch] = {
```

Comment:
Do we need notes about how this will only work on the driver? Or is that implied by it being an RDD.